src/components/Stepper/README.md

```diff
@@ -46,7 +46,7 @@ export default {
 
 ## Events
 
-| Event          | Type     | Description           |
-| -------------- | -------- | --------------------- |
-| stepper:update | `number` | updated stepper value |
+| Event          | Type | Description |
+| -------------- | ---- | ----------- |
+| stepper:update | -    | —           |
```

Comment:
Unfortunate this got removed, but since they're autogenerated I'm not sure how we can put these back in. @pretzelhammer do you know how we should handle event documentation?

Reply:
hmmm, this is already properly documented here, not sure why the auto-generator isn't catching it: https://github.com/square/maker/pull/316/files#diff-1844fa4f25a196f09e947190a8f5d759a44d6a6e0889f6f2905d915e1218e46fR204-R208

Reply:
anyway, fixing this is not a blocker for this PR, so I'm gonna resolve this thread